packages/dart/noports_core/lib/src/sshnp/sshnp_core.dart

```diff
@@ -5,7 +5,6 @@ import 'package:at_client/at_client_mixins.dart';
 import 'package:at_utils/at_logger.dart';
 import 'package:meta/meta.dart';
 import 'package:noports_core/src/common/features.dart';
-import 'package:noports_core/src/common/mixins/apkam_signing.dart';
```

Comment:
If you are uptaking the latest atClient, then you can delete apkam_signing.dart from this repo. It was first created here then upstreamed into at_client in https://github.com/atsign-foundation/at_client_sdk/pull/1546

Reply:
https://github.com/atsign-foundation/noports/pull/2099/commits/71013483af671e53cb07d3d7fc9f6c4be598d321